docs/iidm/exporter/iidm.md

```diff
@@ -106,9 +120,32 @@ The export mode can be:
    - `IidmImportExportMode.ONE_SEPARATED_FILE_PER_EXTENSION_TYPE`: in case we want to export network in a file and each extension type in a separate file.
    Example: if our network `test` has two extensions `loadFoo` and `loadBar` then the network will be exported 
     in the `test.xiidm` file when `loadFoo` and `loadBar` will be exported respectively in `test-loadFoo.xiidm` and `test-loadBar.xiidm`.
-
 The default value for this parameter is `IidmImportExportMode.NO_SEPARATED_FILE_FOR_EXTENSIONS`.
 
+## iidm.export.xml.incremental-conversion
+The `iidm.export.xml.incremental-conversion` property is an optional property that defines whether the export type is 
+incremental or not.
+
+Setting that attribute to true lets us export only the elements having control, state or topology attributes.
+we export those elements to three separated files `base-TOPO.xiidm`, `base-STATE.xiidm` and `base-CONTROL.xiidm`.
+
+The default value for this parameter is `false`.
+
+## iidm.export.incremental.xml.topo
+The `iidm.export.incremental.xml.topo` property is an optional property that defines whether the XIIDM exporter has to
+export elements having topology attributes in a TOPO file or not. 
+
```

Comment:
Use `base-TOPO.xiidm` file instead of TOPO file maybe for the good reference to what you explain previously.